tools/esdlcmd/esdl-publish.cpp

```diff
@@ -79,7 +79,6 @@ class EsdlPublishCmdCommon : public EsdlCmdCommon
                 "   --port <port>                WsESDLConfig service port\n"
                 "   -u, --username <name>        Username for accessing WsESDLConfig service\n"
                 "   -pw, --password <pw>         Password for accessing WsESDLConfig service\n"
-                "   --version <ver>              ESDL service version\n"
```

Comment:
ahh.. ok so then   StringAttr optVersionStr; and double  optVersion; are actually being used as "esdl_definition_version"?? maybe we refactor them to esdl_def_ver?